src/main/java/org/fcrepo/migration/handlers/ocfl/ArchiveGroupHandler.java

```diff
@@ -199,11 +208,17 @@ public void processObjectVersions(final Iterable<ObjectVersionReference> version
                     // Write a file for external content only for plain OCFL migration
                     if (migrationType == MigrationType.PLAIN_OCFL) {
                         content = IOUtils.toInputStream(dv.getExternalOrRedirectURL());
+                        try {
+                            writeDatastreamContent(dv, datastreamHeaders, content, session);
+                        } catch (final IOException io) {
+                            throw new UncheckedIOException(io);
+                        }
```

Comment:
I'm not sure this makes sense. For an external resource, it's just writing the URL for the resource. If there is a digest recorded in F3, would it ever match?

Reply:
thanks - I missed that. I thought it was actually grabbing the content from the url, instead of just writing the URL. Will update.

Reply:
fixed